src/middleware.js

```diff
@@ -4,7 +4,7 @@ import http from './http'
 import { lpApiRequest, lpApiSuccess, lpApiFailure } from './actions'
 
 const DEFAULT_CONFIG_OPTIONS = {
-  onUnauthorized: undefined,
+  onUnauthorized: undefined
```

Comment:
Didn't know that's a thing! Is it just to prevent the chance of adding something and forgetting the comma on the item above?